### DISCUSSION TITLE
Get values of HP-PCU (cooling)

### DISCUSSION THREAD

**Discussion by Pat-MT09** [emsesp/EMS-ESP32#1797] [Q&A]
Hi!

i have got an Bosch CS7800i LW MB (recognized as Enviline/Compress 6000AW/Hybrid 3…) with a passive cooling station Bosch HP-PCU. The cooling station is connected via Can-Bus and has 2 Temperatur Sensors and 2 valve. Is it possible to get those values? In the Menu of the Heatpump these values are readable.

TK1 Vorlauftemperatur Kühlung (I11)
TK2 Soletemperatur Kühlung (I10)
VK1 PKS Mischventil (O10)
VK2 PKS 3-Wege-Ventil (O13)

Thanks you for your help!

**Comment by proddy** [comment]
do you have EMS-ESP running? I think you do. Have a look at https://emsesp.github.io/docs/Troubleshooting/#im-missing-certain-data-from-an-ems-device

**Reply by Pat-MT09** [reply]
Thank you!
That helped, i found
TK1 (0x08      0x489).
Could not find TK2 or the valves yet, but maybe it is not working in the forcing mode of the heat-pump and i have to wait until it is cooling on its own.

**Reply by a-lenkov** [reply]
Hello, 
I have a Buderus HP with the passive cooling and have the following errors. Would you please check and let me know if it possible to configure?

Thanks in advance.

[log 11.txt](https://github.com/user-attachments/files/20635696/log.11.txt)

**Reply by proddy** [reply]
> Hello, I have a Buderus HP with the passive cooling and have the following errors. Would you please check and let me know if it possible to configure?
> 
> Thanks in advance.
> 
> [log 11.txt](https://github.com/user-attachments/files/20635696/log.11.txt)

@a-lenkov you're on a very old version of EMS-ESP, please first upgrade and then look at https://docs.emsesp.org/Troubleshooting/#many-rx-errors on how to fix the connectivity. If you still run into errors, open a new GitHub issue so we can help.
